src/style.css

```diff
@@ -0,0 +1,116 @@
+body {
+  margin: 0;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.search-form-big::placeholder {
+  color: #3d4e61;
+  font-weight: 400;
+}
+
+.search-form-big {
+  
+  height: 70px;
+  /* line-height: 22px; */
+  width: 100%;
+  font-size: 16px;
+  font-weight: 300;
+  font-family: Avenir, serif;
```

Comment:
add a font to the `html` tag and inherit font where needed

Reply:
Thank you. All changes done, but with width "inherit" doesn't work.   I don't know why is it.